src/v1/sources/shopify/webhookTransformations/serverSideUtlis.js

```diff
@@ -1,9 +1,6 @@
 const { isDefinedAndNotNull } = require('@rudderstack/integrations-lib');
 const { constructPayload } = require('../../../../v0/util');
-const {
```

Comment:
Are we mapping any additional fields in the new flow or the old tracker flow is not inline with ecomm spec ?

Reply:
old flow was not inline with the spec, reusing them got us some bugs in the QA, mainly with the data types. Hence separated it out